lib/enumerable/any.rb

```diff
@@ -0,0 +1,12 @@
+# frozen_string_literal: true
+
+module Enumerable
+  def my_any?(arg = nil, &proc)
+    if block_given?
+      my_each { |element| return true if proc.nil? ? proc.call(element) : yield(element) }
+    else
```

Comment:
this method should also work when tested when passing a class as an argument

`[nil, true, 99].any?(Integer)                     #=> true
`

Reply:
This point is already taken care

![image](https://user-images.githubusercontent.com/36519478/68625286-c0908380-04a6-11ea-862a-3d8230010364.png)